static/index.html

```diff
@@ -20,12 +20,29 @@
      <header>
        <div class="container">
            <div class="row">
-         </div>
-       </div>
+               <div class="col-sm-6">
+                   <div class="logo">
+                     <img src="images/Official_logo.jpg">
```

Comment:
As a note, we will need to replace this later with a SVG I think. It would be lighter and better quality.